### DISCUSSION TITLE
Benchmarks

### DISCUSSION THREAD

**Discussion by artnoage** [cline/cline#377] [Feature Requests]
So this is not a request per se, more like a question. Have you tried to apply claude-dev in any of the benchmarks to see how it performs? If there is a pipeline like this ready, I have some ideas I want to try re the system prompt.

**Comment by BradKML** [comment]
This but also RooCode https://liveswebench.ai/ https://github.com/RooCodeInc/Roo-Code/discussions/1614 https://github.com/RooCodeInc/Roo-Code/discussions/2018
